docs/source/architecture/validator_network.rst

```diff
@@ -0,0 +1,208 @@
+*****************
+Validator Network
+*****************
+
+The network layer is responsible for communication between validators in a
+Sawtooth network, including performing initial connectivity, peer discovery,
+and message handling. Upon startup, validator instances begin listening on a
+specified interface and port for incoming connections. Upon connection and
+peering, validators exchange messages with each other based on the rules of a
+gossip or epidemic [#f1]_ protocol.
+
+A primary design goal is to keep the network layer as self-contained as
+possible. For example, the network layer should not need knowledge of the
+payload of application messages, nor should it need application-layer provided
+data to connect to peers or to build out the connectivity of the network.
+Conversely, the application should not need to understand implementation
+details of the network in order to send and receive messages.
+
+Services
+========
+
+The choice of 0MQ provides considerable flexibility in both available
+connectivity patterns and the underlying capabilities of the transport layer
+(IPv4, IPv6, etc.)
+
+We have adopted the 0MQ Asynchronous Client/Server Pattern [#f2]_ which consists
+of a 0MQ ROUTER socket on the ‘server’ side which listens on a provided
```

Comment:
Punctuation: I don't think the word 'server' needs to be in quotes. (Style suggestion, not mandate)